src/mito2/src/memtable/partition_tree.rs

```diff
@@ -562,4 +572,166 @@ mod tests {
         assert!(config.dedup);
         assert_eq!(PartitionTreeConfig::default(), config);
     }
+
+    fn metadata_for_metric_engine() -> RegionMetadataRef {
+        let mut builder = RegionMetadataBuilder::new(RegionId::new(123, 456));
+        builder
+            .push_column_metadata(ColumnMetadata {
+                column_schema: ColumnSchema::new(
+                    "__table_id",
+                    ConcreteDataType::uint32_datatype(),
+                    false,
+                ),
+                semantic_type: SemanticType::Tag,
+                column_id: 2147483652,
+            })
+            .push_column_metadata(ColumnMetadata {
+                column_schema: ColumnSchema::new(
+                    "__ts_id",
```

Comment:
`tsid` instead of `ts_id`. But this doesn't seem to matter.


```suggestion
                    "__tsid",
```